onnxruntime/python/tools/quantization/matmul_4bits_quantizer.py

```diff
@@ -165,20 +264,102 @@ def _process_subgraph(self, graph_stack: List[GraphProto]):
         graph_stack.pop()
         return graph
 
+    def _generate_q4_node_config(self):
+        """Generate weight only quant configuration for nodes."""
+        q4_node_config = {}
+        template_config_q4 = {"bits": 4, "group_size": self.block_size, "scheme": "sym" if self.is_symmetric else "asym"}
+        template_config_fp32 = "fp32"
```

Comment:
>template_config_fp32 [](http://example.com/codeflow?start=8&length=20)

can we make the activation support both fp32 and fp16?

Reply:
Will it work if the activation is fp16?

Reply:
I tried fp16 model. Looks like it doesn't work.

Reply:
hi yufeng,  it doesn't work for fp16 now. If we support it in inc v2.4.1 and will release on 12.29, is it acceptable? Or only fp32 is also acceptable?